config/routes.rb

```diff
@@ -219,8 +219,8 @@
         resources :experience_points_records, only: [:index, :destroy]
         get 'invite' => 'user_invitations#new', on: :collection
         post 'invite' => 'user_invitations#create', on: :collection
-        get 'disburse_experience_points' => 'experience_points_disbursement#new', on: :collection
-        post 'disburse_experience_points' => 'experience_points_disbursement#create',
+        get 'disburse_experience_points' => 'experience_points/disbursement#new', on: :collection
```

Comment:
Yeah, should tidy this up. Would it be better to follow this up with a housekeeping PR?